code/modules/projectiles/guns/shotguns.dm

```diff
@@ -910,3 +910,42 @@ can cause issues with ammo types getting mixed up during the burst.
 /obj/item/weapon/gun/shotgun/pump/t35/nonstandard
 	current_mag = /obj/item/ammo_magazine/internal/shotgun/pump/buckshot
 	starting_attachment_types = list(/obj/item/attachable/stock/t35stock, /obj/item/attachable/angledgrip, /obj/item/attachable/magnetic_harness)
+
+//-------------------------------------------------------
+//THE MYTH, THE GUN, THE LEGEND, THE DEATH, THE ZX
+
+/obj/item/weapon/gun/shotgun/zx76
+	name = "\improper ZX-76 assault shotgun"
+	desc = "The ZX-76 Assault Shotgun, a incredibly rare, double barreled semi-automatic combat shotgun with a twin shot mode. Possibly the unrivaled master of CQC. Has a 9 round internal magazine."
+	icon = 'icons/Marine/gun64.dmi'
+	icon_state = "zx-76"
+	item_state = "zx-76"
+	max_shells = 10 //codex
+	caliber = "12 gauge shotgun shells" //codex
+	load_method = SINGLE_CASING //codex
+	fire_sound = 'sound/weapons/guns/fire/shotgun_light.ogg'
+	current_mag = /obj/item/ammo_magazine/internal/shotgun/scout
+	aim_slowdown = 0.45
+	attachable_allowed = list(
+						/obj/item/attachable/bayonet,
+						/obj/item/attachable/reddot,
+						/obj/item/attachable/verticalgrip,
+						/obj/item/attachable/angledgrip,
+						/obj/item/attachable/gyro,
+						/obj/item/attachable/flashlight,
+						/obj/item/attachable/magnetic_harness,
+						/obj/item/attachable/lasersight,
+						/obj/item/attachable/attached_gun/flamer,
+						/obj/item/attachable/attached_gun/shotgun,
+						/obj/item/attachable/attached_gun/grenade)
```

Comment:
Why would you ever add attached_gun? It's a gimmick gun with only CQC in mind, this doesn't belong there